website/templates/home.mako

```diff
@@ -1,5 +1,30 @@
 <%inherit file="base.mako"/>
 <%def name="title()">Home</%def>
+
+<%def name="content_wrap()">
+    <div class="watermarked">
+            ${self.content()}
+    </div><!-- end watermarked -->
+</%def>
+
+
 <%def name="content()">
-    <h1>Placeholder</h1>
+
+    <div id="osfHome"></div>
+
+</%def>
+
+<%def name="stylesheets()">
+  <link rel="stylesheet" href="/static/css/pages/home-page.css">
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+  <link rel="stylesheet" href="/static/vendor/bower_components/jQuery-ui-Slider-Pips/dist/jquery-ui-slider-pips.css">
```

Comment:
@hmoco Remove this, as it is no longer used. Also remove the library from bower.json